frontend/src/pages/LogsPage/LogsTable/LogDetails.css.ts

```diff
@@ -7,3 +8,15 @@ export const line = style({
 		},
 	},
 })
+
+export const linkButton = style({
```

Comment:
Do we have to override the styling from the component library? Can we move this code into the component library if we feel like the styling is important?
cc @ccschmitz on what we should be doing here.

Reply:
It looks like this is a standard secondary tag in the mocks. I'd be curious to know why this doesn't work the way we want it to in this context. Maybe we need to update the styles for the `kind="secondary"` + `emphasis="medium"` compound variant?

https://github.com/highlight/highlight/blob/24a224728391315e9a374e68fd1dd44bf7f9c00a/packages/ui/src/components/Tag/styles.css.ts#L214-L236

Reply:
updated the compound tag style